person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    @age >= 18 || @parent_permission ? true : false
```

Comment:
- Optional
Since you already have `of_age?` method described to check for the age, I would recommend using that method here.

Reply:
- [ ] Great job using the `of_age` method already established, this will make your code more clean 👍🏽